.github/workflows/snap.yaml

```diff
@@ -51,7 +51,10 @@ on:
         required: false
         type: boolean
         default: true
-
+    secrets:
+      store-login:
+        description: "Snap Store credential exported with snapcraft export-login"
+        required: true
```

Comment:
This is only required if the `publish` option is true. I don't know if this can be conditioned, if not I'll suggest to default to false.